lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -3,6 +3,7 @@
 Tests of verify_student views.
 """
 import json
+from opaque_keys.edx.keys import UsageKey
```

Comment:
Please move this import to external apps import, since `opaque_keys` code lives in external repo.